tests/util/test_uuid.py

```diff
@@ -0,0 +1,11 @@
+"""Test Home Assistant uuid util methods."""
+
+import uuid
+
+import homeassistant.util.uuid as uuid_util
+
+
+async def test_uuid_v1mc_hex():
+    """Verify we can generate a uuid_v1mc and return hex."""
+    assert len(uuid_util.uuid_v1mc_hex()) == 32
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6063052)